pkg/gui/files_panel.go

```diff
@@ -256,7 +248,7 @@ func (gui *Gui) handleFileOpen(g *gocui.Gui, v *gocui.View) error {
 	if err != nil {
 		return err
 	}
-	return gui.genericFileOpen(g, v, file.Name, gui.OSCommand.OpenFile)
+	return gui.genericFileOpen(g, v, file.Name, func(filename string) (*exec.Cmd, error) { return nil, gui.OSCommand.OpenFile(filename) })
```

Comment:
Perhaps we should not use genericFileOpen for gui.OSCommand.OpenFile, and we should just give it its own gui function i.e.

```
func (gui *Gui) openFile(filename string) error {
	err := gui.OSCommand.OpenFile(filename)
	if err != nil {
		return gui.createErrorPanel(gui.g, err.Error())
	}
	return nil
}
```

That way we don't need to duplicate code when it comes to the anonymous function being used here